controllers/storagecluster/cephcluster.go

```diff
@@ -453,7 +445,7 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 			},
 			Storage: rookCephv1.StorageScopeSpec{
 				StorageClassDeviceSets:       newStorageClassDeviceSets(sc),
-				Store:                        osdStore,
+				Store:                        rookCephv1.OSDStore{},
```

Comment:
Can not we just remove this field instead of having an empty struct?

Reply:
removed.